docs/topics/rn-known-issues.adoc

```diff
@@ -18,21 +18,17 @@ For a complete list of all known issues, see the list of link:https://issues.red
 
 |link:https://issues.redhat.com/browse/WINDUP-3138[WINDUP-3138]
 |CodeReady Workspace plugin
-|The plugin does not work with CodeReady Workspace 2.10.
+|The plugin works with CodeReady Workspace 2.09, but not version 2.10.
 
-|link:https://issues.redhat.com/browse/WINDUP-3019[WINDUP-3019]
-|CLI
-|Bash completion feature displays an `mta-cli not found` error.
+|link:https://issues.redhat.com/browse/WINDUP-3165[WINDUP-3165]
+|Web console
+|MTA analysis fails on OpenShift Container Platform when web console custom labels are uploaded and advanced options are enabled.
```

Comment:
What happened to my comment on this line?

Reply:
Oh, I see. "web console custom labels" is not a good idea because there is also an OpenShift web console

Reply:
Change 'web console' to MTA